src/styles.css

```diff
@@ -4,4 +4,6 @@
     background-color: #21D4FD;
     background-image: linear-gradient(19deg, #21D4FD 0%, #B721FF 100%);
     color: white;
-} */
\ No newline at end of file
+} */
+
+body{overflow-x: hidden;}
```

Comment:
instead of fixing the style, try removing the padding from the faulty <section>

Reply:
i tried to the same but it isn't working for some reason. i think we don't need horizontal scroll on our page anyway.